src/components/input/Select.tsx

```diff
@@ -46,6 +46,9 @@ export type SelectOptionProps<T> = {
     | ComponentChildren
     | ((status: SelectOptionStatus) => ComponentChildren);
   classes?: string | string[];
+
+  /** Ref associated with outermost option element */
```

Comment:
"outermost" to me implies there will be nested `option` elements. I think what you mean to say is that this will be applied to the container element that has the semantic role `option` and is focusable. The "focusable" property is something that is important for our current use case.

Reply:
Yes, that's what I meant. I'll reword it.

Reply:
Or perhaps it should have been `Ref associated with the option's outermost element`, but I think the "container" keyword is less confusing.